### PR TITLE
add c from f & up docs contribute

### PR DESCRIPTION
**Pull request by lazarusA**
- add example on how to create a data cube given a function that acts on space and time
- some tweaks to docs
- adds guide for contributors to docs

### CODE REVIEW

File: docs/examples/UserGuide/create_from_func.jl [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# ## Wrap function for mapCube output
+
+function g(xout,lo,la,t)
+    xout .= f.(lo,la,t)
+end
+
+# Note the applied `.` after `f`, this is because we will slice across time,
+# hence the application is broadcasted along this dimension.
+
+# ## Create Cube's Axis
```
Comment (felixcremer) [inline docs/examples/UserGuide/create_from_func.jl:17]:
```suggestion
# ## Create Cube's Axes
```

File: docs/examples/UserGuide/create_from_func.jl [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    indims = (InDims(), InDims(), InDims("time")),
+    outdims = OutDims("time", overwrite=true,
+    path = "my_gen_cube.zarr", backend=:zarr, outtype=Float32),
+    #max_cache=1e9
+    )
+
+# !!! warning "time axis is first"
+#     Note that currently the `time` axis in the output cube goes first.
+
+# Check that is working
```
Comment (felixcremer) [inline docs/examples/UserGuide/create_from_func.jl:40]:
```suggestion
# Check that it is working
```

File: docs/examples/contribute/contribute.jl [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+# `docs> julia`
+
+# `julia> ]`
+
+# `(docs) pkg> activate .`
+
+# Next, run the scripts:
+# !!! info
+#     Generate files and build docs by running:
```
Comment (felixcremer) [inline docs/examples/contribute/contribute.jl:35]: Does this mean include("genfiles.jl") or should I do 'julia genfiles.jl'?
Maybe you could clarify this here.

**Review by felixcremer — COMMENTED** [review]
Thanks, the example looks good for me.  I did not check the changes to mkdocs.yml because you would know what you are doing.

**Comment by felixcremer** [comment]
What is up with the Documenter CI? Can you see, why this is failing to build the documentation?

**Comment by lazarusA** [comment]
@meggart the documenter key is not working. @felixcremer

File: docs/examples/contribute/contribute.jl (hunk 3 above)
Comment (lazarusA) [reply docs/examples/contribute/contribute.jl:35]: from the repl is include("genfiles.jl") but usually just select all lines and do the run interactive.

File: docs/examples/UserGuide/create_from_func.jl [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+# ## Generate Cube from function
+# The following generates a new `cube` using `mapCube` and saving the output directly to disk.
+
+gen_cube = mapCube(g, (lon, lat, time);
+    indims = (InDims(), InDims(), InDims("time")),
+    outdims = OutDims("time", overwrite=true,
+    path = "my_gen_cube.zarr", backend=:zarr, outtype=Float32),
+    #max_cache=1e9
+    )
```
Comment (meggart) [inline docs/examples/UserGuide/create_from_func.jl:35]:
```julia
gen_cube = mapCube(g, (lon, lat, time);
    indims = (InDims("lon"), InDims(), InDims()),
    outdims = OutDims("lon",overwrite=true,
    path = "my_gen_cube.zarr", backend=:zarr, outtype=Float32),
    #max_cache=1e9
    )
```

**Pull request merged**